src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1660,9 +1660,9 @@ void program_node::create_onednn_primitive_attributes(
                                 update_onednn_post_op_list(onednn_post_op_type::eltwise_linear, empty_mem);
                             } else {
                                 auto in_shift = get_input_layout(dep_idx++);
-                                dnnl::memory::desc in_shift_desc = onednn::layout_to_memory_desc(in_shift, dnnl::memory::format_tag::ab, true);
+                                dnnl::memory::desc in_shift_desc = onednn::layout_to_memory_desc(in_shift, onednn::get_default_data_format(in_shift), false);
```

Comment:
could you remove last argument `false`?

Reply:
Yes, I'll update it as you reviewed. Thank you.